odoo/tools/convert.py

```diff
@@ -498,38 +496,65 @@ def _tag_template(self, el):
 
         record = etree.Element('record', attrib=record_attrs)
         record.append(Field(name, name='name'))
-        record.append(Field(full_tpl_id, name='key'))
         record.append(Field("qweb", name='type'))
-        if 'track' in el.attrib:
-            record.append(Field(el.get('track'), name='track'))
+
+        View = self.env['ir.ui.view']
+
+        if 'track' in View:
+            if 'track' in el.attrib:
+                record.append(Field(name='track', eval=el.get('track')))
+            else:
+                record.append(Field(name="track", eval="False"))
+
         if 'priority' in el.attrib:
             record.append(Field(el.get('priority'), name='priority'))
+
         if 'inherit_id' in el.attrib:
             record.append(Field(name='inherit_id', ref=el.get('inherit_id')))
-        if 'website_id' in el.attrib:
-            record.append(Field(name='website_id', ref=el.get('website_id')))
+            if el.get('primary') == 'True':
+                # Pseudo clone mode, we'll set the t-name to the full canonical xmlid
+                el.append(
+                    builder.E.xpath(
+                        builder.E.attribute(full_tpl_id, name='t-name'),
+                        expr=".",
+                        position="attributes",
+                    )
+                )
+                record.append(Field('primary', name='mode'))
+            else:
+                record.append(Field("extension", name="mode"))
+        else:
+            record.append(Field(name="inherit_id", eval="False"))
+            record.append(Field('primary', name='mode'))
+
+        if 'website_id' in View:
+            if 'website_id' in el.attrib:
+                record.append(Field(name='website_id', ref=el.get('website_id')))
+            # else:
+            #     Keep default value (default website)
+            #     record.append(Field(name="website_id", eval="False"))
+
         if 'key' in el.attrib:
             record.append(Field(el.get('key'), name='key'))
-        if el.get('active') in ("True", "False"):
-            view_id = self.id_get(tpl_id, raise_if_not_found=False)
-            if self.mode != "update" or not view_id:
+        else:
+            record.append(Field(full_tpl_id, name='key'))
```

Comment:
Keys are only used for qweb templates.
(We're considering separating the ir.ui.view template into a table for ir.ui.view and an ir.qweb table.)